sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -65,8 +65,7 @@ public async Task VerifyImdsRequestMockAsync()
         [Test]
         public async Task VerifyImdsRequestWithClientIdMockAsync()
         {
-            using (new TestEnvVar("MSI_ENDPOINT", null))
-            using (new TestEnvVar("MSI_SECRET", null))
+            using (new TestEnvVar(new Dictionary<string, string> { { "MSI_ENDPOINT", null }, { "MSI_SECRET", null } }))
```

Comment:
I don't know if you care but you can add an `Add` method to `TestEnvVar` and get a bit cleaner

```
new TestEnvVar(){ { "MSI_ENDPOINT", null }, { "MSI_SECRET", null } }
```
syntax.

Reply:
Or maybe use a target-typed new